src/components/List/List.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+  <div :id="id || `list-${_uid}`">
+    <div v-for="(items, index) in itemsList" :key="index" class="sol-list">
+      <div class="list">
+        {{ items.list }}
+      </div>
+      <div class="list-item">
+        {{ items.listItem }}
+      </div>
+    </div>
+  </div>
```

Comment:
Pesando em algo mais semântico, poderíamos ir com uma lista de definição, exemplo.

```suggestion
  <dl :id="id || `list-${_uid}`">
    <template v-for="(items, index) in itemsList">
      <dt :key="`title-${index}`">{{ items.list }}</dt>
      <dd :key="`title-${index}`">{{ items.listItem }}</dd>
    </template>
  </dl>
```